manager/controlapi/service.go

```diff
@@ -161,6 +166,22 @@ func validateContainerSpec(taskSpec api.TaskSpec) error {
 	return nil
 }
 
+// validateExtraHost validates that the specified string is a valid extrahost and returns it.
+// ExtraHost is in the form of name:ip where the ip has to be a valid ip (IPv4 or IPv6).
+func validateExtraHost(hosts []string) error {
+	for _, host := range hosts {
+		arr := strings.Split(host, ":")
```

Comment:
This wouldn't work for `node1:[1fff:0:a88:85a3::ac1f]`. I think it should be `	arr := strings.SplitN(host, ":", 2)
`